pkg/component/watch_test.go

```diff
@@ -1,3 +1,6 @@
+// +build !windows
```

Comment:
Skip build for windows and macOS. Watch test will be revised in https://github.com/openshift/odo/issues/2570 for windows and macOS.